docs/components/Balance.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 6
+---
+
+# Balance
+
+Displays the balance of a given address in both ether (ETH) and US dollars (USD).
+
+![Balance Example](/img/Balance.png)
+
+## Import
+
+```ts
+import { Balance } from "~~/components/scaffold-eth";
+```
+
+## Usage
+
+```ts
+<Balance
+  address={address}
+  className="text-3xl bg-neutral h-14 p-6 rounded-md mb-6"
```

Comment:
Umm not sure if we should set `className` for here since its optional field...What we mention in `props` section and remove it here ?  : 


- `className` => An optional prop to pass additional CSS styling to the component. You can use tailwind / daisyUI  classes like `text-3xl` etc ?

Reply:
Great suggestions, just updated the example to get rid of className and update the prop description ♥

New prop description:
`className` (optional) => Prop to pass additional CSS styling to the component. You can use tailwind / daisyUI classes like `text-3xl` for styling.